BuildScript.Nuke/ICleanup.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Nuke.Common;
+using Nuke.Common.IO;
+
+public interface ICleanup : INukeBuild
+{
+  public Target CleanFolders => _ => _
+      .Executes(() =>
+      {
+        var directoryCollection = new DirectoryCollection(RootDirectory);
+        FileSystemTasks.DeleteDirectory(directoryCollection.OutputDirectory);
+        FileSystemTasks.DeleteDirectory(directoryCollection.LogDirectory);
+        FileSystemTasks.DeleteDirectory(directoryCollection.TempDirectory);
+      });
+}
```

Comment:
Please implement as normal target, splitting this into components will be a task at a later point.